javatests/google/registry/flows/EppFlowInjectionModule.java

```diff
@@ -0,0 +1,46 @@
+package google.registry.flows;
+
+import org.joda.time.DateTime;
+
+import dagger.Module;
+import dagger.Provides;
+import google.registry.flows.EppException.UnimplementedOptionException;
+import google.registry.flows.domain.ExtraDomainValidation;
+import google.registry.model.domain.launch.LaunchCreateExtension;
+import google.registry.model.registry.label.ReservationType;
+
+@Module
+public class EppFlowInjectionModule {
```

Comment:
I'd suggest renaming this class in a way that makes it clear it's for testing.  How about TestingFlowInjectionModule?